tools/sdk-generation-pipeline/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN apt install python3-pip -y && apt install python3-venv -y && pip3 install --
 RUN add-apt-repository ppa:deadsnakes/ppa -y && apt-get install python3.10 -y && apt-get install python3.10-venv -y
 
 # install powershell
-RUN wget -q https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb
+RUN wget -q https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb
```

Comment:
Please revert it because our docker image is based on ubuntu 20.04 now.

Reply:
@dw511214992 I have reverted the change.

- Could we move the docker image to Ubuntu 22.04 ? Could I somehow assist with it?
- Please note I made few more updates to Ubuntu 22.04: in [tools/sdk-generation-pipeline/ci.yml](https://github.com/Azure/azure-sdk-tools/pull/4996/files#diff-39c24c7d0a919c500c601a7b015d4e7e48589b1528bd68abc51c2f79e4f9ae96) and [tools/sdk-generation-pipelines/sdk-generation-pipeline.yml](https://github.com/Azure/azure-sdk-tools/pull/4996/files#diff-8001a59ebe4bc27e2cbc2eff492b5f4dda2ddd95f67ab746d445d189aaae1de2). Are you OK with these changes?
- I also see few usages of `ubuntu-latest`, e.g. [in tools/sdk-generation-pipeline/.github/workflows/ci.yml](https://github.com/Azure/azure-sdk-tools/blob/dd3c489da0917fc9efd19deeb20f31406f3a5a42/tools/sdk-generation-pipeline/.github/workflows/ci.yml). I also plan to update it to `Ubuntu 22.04` to avoid issues like #4888.

Reply:
@konrad-jamrozik  I have added comment to your PR. thanks